CHANGELOG.asciidoc

```diff
@@ -36,6 +36,7 @@ This release also includes changes from <<release-3-2-10, 3.2.10>>.
 
 ==== Bugs
 
+* TINKERPOP-2081: Fix PersistedOutputRDD to eager persist RDD
```

Comment:
could you please move this to the 3.3.5 section - 3.3.4 is already released. also, you don't need to add it in the "Bugs" section with JIRA issue...we generate that on release. You can just add a bullet with a description of the change.

Reply:
fixed.